packages/ui-components/src/components/UIComboBox/UIComboBox.tsx

```diff
@@ -693,6 +693,10 @@ export class UIComboBox extends React.Component<UIComboBoxProps, UIComboBoxState
             autofill.readOnly = true;
             autofill.tabIndex = tabIndex;
         }
+        // This is a workaround for FluentUI not handling aria-invalid correctly for a ComboBox.
+        // For the time being we add logic here to set aria-invalid based on the presence of
+        // an errorMessage.
+        autofill['aria-invalid'] = this.props.errorMessage !== undefined;
```

Comment:
@Thomas-Neuhauser 

I just checked if we pass empty string, then visually we do not see error message, but this code will add `aria-invalid` as `true`.
![image](https://github.com/user-attachments/assets/0a06aa40-46b5-429c-a8d2-7d10d708dfa4)

Also checked `TextField` - if empty `errorMessage` is passed, then `aria-invalid`is `false`
![image](https://github.com/user-attachments/assets/e37b3892-6dd8-4613-97e7-1ccaed4f2f9d)


Does it makes sense in this case to add `aria-invalid` if empty error message is passed? I think if empty `errorMessage` is passed, then we should set `aria-invalid` as `false` like it is for UITextInput

Reply:
Hello Andis, ok, the empty string case is a good one, I will extend the logic to cover that.
Setting aria-invalid = false is correct from my point of view, since this replicates the behavior other FluentUI controls already show.
You can check https://developer.microsoft.com/en-us/fluentui#/controls/web/dropdown and scroll down a little, there's an example where you can switch an error message on and off and if you check the render output, there's always an aria-invalid attribute either true or false.
I assume that screenreaders react to this change and I wanted to replicate this behavior.
I'll add a check for an empty string.

Reply:
@Thomas-Neuhauser 

my 'complain' was only about empty `errorMessage`, otherwise looks ok